CONTRIBUTING.md

```diff
@@ -54,6 +54,21 @@ translation PRs are opened within a week of the original newsletter being
 published for new newsletters. That said, we also encourage translation of
 older newsletters and blog posts as well.
 
+### Reviewing changes to the website generation code
+
+When reviewing changes to the Jekyll code that generates the website, it's useful to compare
+the generated website before and after the change. To do so, you can run the following commands:
+
+```bash
+git checkout $reviewbranch  # Checkout the branch that you want to review
+rm -rf _site.bak  # Remove the backup site directory if it exists
+JEKYLL_ENV=local make build  # Do a clean build of the site to the _site directory. The local env is to remove the 'updated' field which causes irrelevant diffs
+mv _site _site.bak  # Rename the generated site directory
+git checkout master  # Checkout the base branch
+JEKYLL_ENV=local make build  # Do a clean build of the site to the _site directory
+diff _site _site.bak  # Compare the generated sites
```

Comment:
My diff (GNU diffutils) doesn't automatically diff subdirs.  E.g.:

    $ cd /tmp
    $ mkdir -p a/b c/b
    $ echo foo > a/b/foo.txt
    $ diff a c
    Common subdirectories: a/b and c/b

It also doesn't automatically display the diffs of files that only exist in one version (it just prints a line about "Only in c: foo.txt", which is easy to miss in a big diff).

For those reasons, I typically use this diff command: `diff --recursive --newfile` (or `diff -rN`).  (Personally, I also use `-u` for unified diffs, which makes the nice mnemonic `-ruN`)

Reply:
Much better. I usually use meld locally for running diffs and I wanted to put `diff` here to make the instructions universal. The options you suggest are good suggestions.